org/views.py

```diff
@@ -54,6 +54,50 @@ def post(self,request):
             data = serializer.errors
             return Response(data, status.HTTP_400_BAD_REQUEST)
 
+class GroupView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    @swagger_auto_schema(
+        operation_id='create_group',
+        request_body=CreateGroupSerializer,
+        responses={
+            '201': set_example({}),
+            '400': set_example(responses.create_group_400),
+            '401': set_example(responses.create_group_401),
+            '403': set_example(responses.create_group_403),
+            '404': set_example(responses.create_group_404)
+        },
+    )
+    def post(self,request, org_id):
+        """
+        When a user posts the required details to this API, a number of checks follow:
```

Comment:
General explanation of a function can be given here : i.e What does this function do.
Other things can be commented over with code, rather that commenting everything under the function.

Reply:
But still, like we just want to know, what happens in this function, we don't want to know what test happen or what cases we have inside a function right. As only developer needs to see those, they obviously see that in code (comments). And Documentation on the other hand should be kept less technical.

Even what Ansh has mentioned here are kind of possible responses of API, which I think doesn't qualify as a good function description. 🤔 What say ? @naveennvrgup  @anshsrtv @priyanshisharma

Reply:
Well said sir🤣🤣